src/main/java/net/coderbot/iris/mixin/MixinMinecraftClient.java

```diff
@@ -0,0 +1,51 @@
+package net.coderbot.iris.mixin;
+
+import net.coderbot.iris.Iris;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.client.render.WorldRenderer;
+import net.minecraft.text.TranslatableText;
+import net.minecraft.util.Formatting;
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(MinecraftClient.class)
+public abstract class MixinMinecraftClient {
+    @Shadow @Nullable public ClientPlayerEntity player;
+
+    @Shadow @Final public WorldRenderer worldRenderer;
+
+    @Inject(method = "tick", at = @At("TAIL"))
+    private void iris$tickKeybind(CallbackInfo ci) {
```

Comment:
`fabric-lifecycle-events-v1` provides a ClientTickCallback, that would probably be a better choice here.